### DISCUSSION TITLE
百度网盘下载报错！

### DISCUSSION THREAD

**Discussion by sjkhsl** [AlistGo/alist#773] [Q&A]
点击下载后页面报错，{"error_code":31362,"error_msg":"sign error","error_info":"","request_id":579299773434499552}
怎么解决？

**Comment by youtehub** [comment]
WebDAV 策略
302 重定向：重定向到真实链接
使用代理 URL：重定向到代理 URL
本机代理：直接通过本地中转返回数据（最佳兼容性） 
感觉应该是 WebDAV 策略选择错误导致的

**Comment by ohmyarch** [comment]
一样

**Comment by xjl456852** [comment]
一样的问题,用的最新版
